config/routes.rb

```diff
@@ -25,7 +25,7 @@ def matches?(request)
         get '/authn-jwt/:service_id/:account/status' => 'authenticate#authn_jwt_status'
         get '/:authenticator(/:service_id)/:account/status' => 'authenticate#status'
 
-        patch '/:authenticator/:service_id/:account' => 'authenticate#update_config'
+        patch '/:authenticator(/:service_id)/:account' => 'authenticate#update_config'
```

Comment:
WDYT on the following change?
```
patch '/:authn-gcp/:account' => 'authenticate#update_config'
patch '/:authenticator/:service_id/:account' => 'authenticate#update_config'
```

Reply:
Think that there are some other authenticator define with service-id as an optional parameter